### PR TITLE
Add docs for Tempo search

### PR DESCRIPTION
**Pull request by zoltanbedi**
**What this PR does / why we need it**:

Adds doc to Tempo search feature.

Merge https://github.com/grafana/website/pull/4510 once this pr is merged.

### CODE REVIEW

File: docs/sources/datasources/tempo.md [hunk 1]
```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 ## Query traces
 
 You can query and display traces from Tempo via [Explore]({{< relref "../explore/_index.md" >}}).
-To query a particular trace, insert its trace ID into the query text input.
+You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. In order to have search results you need to have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that points to this data source.
```
Comment (oddlittlebird) [inline docs/sources/datasources/tempo.md:40]:
```suggestion
You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.
```

File: docs/sources/datasources/tempo.md [hunk 2]
```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
...
 
-{{< docs-imagebox img="/img/docs/v73/tempo-query-editor.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor" >}}
+{{< docs-imagebox img="/img/docs/tempo/query-editor-search.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor showing the search tab" >}}
+
+To query a particular trace, select TraceID query type and insert the ID into the Trace ID field.
```
Comment (oddlittlebird) [inline docs/sources/datasources/tempo.md:44]:
```suggestion
To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.
```

**Comment by zoltanbedi** [comment]
@grafana/docs-squad can I get another round of review here please?

File: docs/sources/datasources/tempo.md [hunk 3]
```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
...
 You can query and display traces from Tempo via [Explore]({{< relref "../explore/_index.md" >}}).
-To query a particular trace, insert its trace ID into the query text input.
+You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.
 
-{{< figure src="/static/img/docs/v73/tempo-query-editor.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor" >}}
+{{< figure src="/static/img/docs/tempo/query-editor-search.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor showing the search tab" >}}
+
+To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.
+
+{{< figure src="/src/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}
```
Comment (robbymilo) [inline docs/sources/datasources/tempo.md:46]:
```suggestion
{{< figure src="/static/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}
```

Assuming you want the image from https://github.com/grafana/website/pull/4510.

Comment (zoltanbedi) [reply docs/sources/datasources/tempo.md]: yes thank you, I missed this one

File: docs/sources/datasources/tempo.md [hunk 4]
```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 ## Query traces
 
 You can query and display traces from Tempo via [Explore]({{< relref "../explore/_index.md" >}}).
-To query a particular trace, insert its trace ID into the query text input.
+You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.
```
Comment (osg-grafana) [inline docs/sources/datasources/tempo.md:40]:
```suggestion
You can search for traces if you set up the trace to logs setting in the data source configuration page. To find traces to visualize, use the [Loki query editor]({{< relref "loki.md#loki-query-editor" >}}). To get search results, you must have [derived fields]({{< relref "loki.md#derived-fields" >}}) configured, which point to this data source.
```

File: docs/sources/datasources/tempo.md [hunk 5]
```diff
@@ -37,9 +37,13 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
...
 
-{{< figure src="/static/img/docs/v73/tempo-query-editor.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor" >}}
+{{< figure src="/static/img/docs/tempo/query-editor-search.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor showing the search tab" >}}
+
+To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.
```
Comment (osg-grafana) [inline docs/sources/datasources/tempo.md:44]:
```suggestion
To query a particular trace, select the **TraceID** query type, and then put the ID into the Trace ID field.
```

**Review by osg-grafana — APPROVED** [review]
Approving to unblock for 8.0-beta3, and please incorporate the review comments.

The folks in the US will not be online for several hours, hence my review.

**Pull request merged**